docs/MO_DG/prepare_model/Getting_performance_numbers.md

```diff
@@ -1,28 +1,28 @@
 # Getting Performance Numbers {#openvino_docs_MO_DG_Getting_Performance_Numbers}
 
 
-## Tip 1. Measure the Proper Set of Operations 
+## Tip 1: Measure the Proper Set of Operations 
 
-When evaluating performance of your model with the OpenVINO Runtime, you must measure the proper set of operations. To do so, consider the following tips: 
+When evaluating performance of model with the OpenVINO Runtime, it is required to measure the proper set of operations. To do so, consider the below options: 
 
 - Avoid including one-time costs like model loading.
 
-- Track separately the operations that happen outside the OpenVINO Runtime, like video decoding. 
+- Track separately operations which occurred outside the OpenVINO Runtime (like video decoding). 
 
-> **NOTE**: Some image pre-processing can be baked into the IR and accelerated accordingly. For more information, refer to [Embedding the Preprocessing](Additional_Optimizations.md). Also consider [Runtime Optimizations of the Preprocessing](../../optimization_guide/dldt_deployment_optimization_common).
+> **NOTE**: Some image pre-processing can be baked into the IR and accelerated accordingly. For more information, see the [Embedding the Preprocessing guide](Additional_Optimizations.md). Additional information are also present in the [Runtime Optimizations of the Preprocessing guide](../../optimization_guide/dldt_deployment_optimization_common.md).
 
-## Tip 2. Getting Credible Performance Numbers 
+## Tip 2: Getting Credible Performance Numbers 
 
-You need to build your performance conclusions on reproducible data. Do the performance measurements with a large number of invocations of the same routine. Since the first iteration is almost always significantly slower than the subsequent ones, you can use an aggregated value for the execution time for final projections:
+Performance conclusions should be build upon reproducible data. As for the performance measurements, they should be done with a large number of invocations of the same routine. Since the first iteration is almost always significantly slower than the subsequent ones, you can use an aggregated value for the execution time for final projections:
```

Comment:
```suggestion
Performance conclusions should be build upon reproducible data. As for the performance measurements, they should be done with a large number of invocations of the same routine. Since the first iteration is almost always significantly slower than the subsequent ones, use an aggregated value for the execution time for final projections:
```